lib/sucker_punch/queue.rb

```diff
@@ -39,7 +39,8 @@ def registered?
     end
 
     def name
-      klass.to_s.underscore.to_sym
+      klass_name = klass.to_s.underscore
+      "#{SuckerPunch::NAME_PREFIX}_#{klass_name}".to_sym
```

Comment:
Is this the right place to be doing this? Does the user need to know that SuckerPunch job names are being given a prefix in the Celluloid registry?